wd_helper.au3

```diff
@@ -2098,9 +2098,9 @@ Func __WD_JsonElement($sElement)
 EndFunc   ;==>__WD_JsonElement
 
 ; #FUNCTION# ====================================================================================================================
```

Comment:
Also could I ask you about filling out this function header.
As this was your concept and I'm not so good in English.